### DISCUSSION TITLE
Adding a page

### DISCUSSION THREAD

**Discussion by sniperfull** [crater-invoice-inc/crater#793] [Q&A]
Hello, i started using Craterapp and love it, although i would like to add a custom page, that has some things specific for my company. But i am wondering what the 'correct' way is to add a new page so that it is to some extent implemented the same as the rest. Would really love some help on this!

**Comment by mohitpanjwani** [comment]
Hey @sniperfull, Thanks for appreciating. 

You can do that using custom templates. Please read the guide to learn more about how to create your own custom template: https://docs.craterapp.com/guide/custom-templates.html
